### DISCUSSION TITLE
Toggle component style is broken when on hover and on focus

### DISCUSSION THREAD

**Discussion by naufal3737** [saadeghi/daisyui#2451] [Q&A] [answered]
Hello, I'm new in programming.
I want to use the toggle component for remember me but it seems broken when I hover and click it.
I use Laravel.

![image](https://github.com/saadeghi/daisyui/assets/61531022/27cf0fe4-ebc0-433a-9b2d-348f0b4c1883)
![image](https://github.com/saadeghi/daisyui/assets/61531022/ee7b12c8-0b7e-4e99-bc5d-588648246489)

I already saw the FAQ and tried removing tailwindcss/forms but the problem still exist

**Comment by saadeghi** [comment, marked as answer]
Do you have any other CSS code in your project that may apply styles to inputs?   

Can you share a small reproduction repo so I can reproduce the problem? Or if the project is online, a link would be enough so I can see the styles.

**Reply by naufal3737** [reply]
It's actually in my css code that change this behavior, thank you very much sorry to bother with such a newbie question.

I think it's this one that messed up the toggle style.

`input:active,
input:focus,
input:hover,
input:visited,
input.active,
input.focus,
input.hover,
input.visited {
    box-shadow: unset !important;
    outline: 0 !important;
}`

**Comment by EvgeniiaVak** [comment]
I had the same issue, the problem was I had [tailwindcss-forms](https://github.com/tailwindlabs/tailwindcss-forms) installed for some reason, and they were conflicting.

**Reply by RealPresidentlin** [reply]
Thank you!!!
